package.json

```diff
@@ -57,7 +57,13 @@
   },
   "dependencies": {
     "@madie/madie-models": "^1.3.11",
-    "axios": "^0.27.2",
+    "@types/jest": "^27.0.1",
+    "@types/react": "^17.0.19",
+    "@types/react-dom": "^17.0.9",
+    "@types/systemjs": "^6.1.1",
+    "@types/webpack-env": "^1.16.2",
```

Comment:
not sure why this is required? is it because of axios update? Also, these are dev-dependecies

Reply:
@adongare I'm not sure but it gets added after the axios update. So should I keep them or remove them?

Reply:
@adongare I removed them

Reply:
I think we can remove. it is already there in dev-dependencies